docker-compose-aws.yml

```diff
@@ -0,0 +1,94 @@
+# Development deployment using AWS RSTUF Services
+version: "3.9"
+
+volumes:
+  repository-service-tuf-storage:
+  repository-service-tuf-api-data:
+  repository-service-tuf-redis-data:
+  repository-service-tuf-pgsql-data:
+
+services:
+  postgres:
+    image: postgres:15.1
+    ports:
+      - "5433:5432"
+    environment:
+      - POSTGRES_PASSWORD=secret
+    volumes:
+      - "repository-service-tuf-pgsql-data:/var/lib/postgresql/data"
+    healthcheck:
+      test: ["CMD", "pg_isready", "-U", "postgres", "-d", "postgres"]
+      interval: 1s
+
+  repository-service-tuf-api:
+    image: ghcr.io/repository-service-tuf/repository-service-tuf-api:${API_VERSION}
+    volumes:
+      - repository-service-tuf-api-data:/data
+    ports:
+      - 80:80
+    environment:
+      - RSTUF_BROKER_SERVER=redis://redis
+      - RSTUF_REDIS_SERVER=redis://redis
+    depends_on:
+      redis:
+        condition: service_healthy
+
+  web:
+    image: python:3.10-slim-buster
+    command: python -m http.server -d /var/opt/repository-service-tuf/storage 8080
+    volumes:
+      - repository-service-tuf-storage:/var/opt/repository-service-tuf/storage
+    ports:
+      - "8080:8080"
+
+  redis:
+    image: redis:4.0
+    volumes:
+      - repository-service-tuf-redis-data:/data
+    ports:
+      - "6379:6379"
+    healthcheck:
+      test: ["CMD", "redis-cli", "ping"]
+      interval: 1s
+
+  localstack:
+    image: localstack/localstack:2.2
+    environment:
+      HOSTNAME: "localstack"
+      LOCALSTACK_HOST: "localstack"
+      LS_LOG: "error"
+    ports:
+      - "4566:4566"
+    volumes:
+      - "./tests/files/aws/init-services.sh:/etc/localstack/init/ready.d/init-services.sh"
+
+  repository-service-tuf-worker:
+    build:
+      context: .
+    entrypoint: "bash entrypoint-dev.sh"
+    environment:
+      - DATA_DIR=./data
+      - RSTUF_STORAGE_BACKEND=AWSS3
+      - RSTUF_AWSS3_STORAGE_BUCKET=tuf-metadata
+      - RSTUF_AWSS3_STORAGE_ACCESS_KEY=access_key
+      - RSTUF_AWSS3_STORAGE_SECRET_KEY=secret_key
+      - RSTUF_AWSS3_STORAGE_REGION=us-east-1
+      - RSTUF_AWSS3_STORAGE_ENDPOINT_URL=http://localstack:4566
```

Comment:
Do we want to try and test without the optional values to continuously test that RSTUF works without user input for them?